### PR TITLE
feat: use tracking section data from api

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[會員制] 重要報導追蹤-串接](https://app.asana.com/0/1203699264568808/1207085276870887/f)

# Notice

# Dependency
[npm-packages#562](https://github.com/twreporter/twreporter-npm-packages/pull/562)

### CODE REVIEW

File: src/components/my-reading/article-tracking-card.js [hunk 1]
```diff
@@ -167,12 +168,19 @@ export const ArticleTrackingCard = ({
   trackingTitle,
   trackingContent,
   trackingArticleTitle,
+  trackingArticleSlug,
 }) => {
+  const { releaseBranch } = useContext(CoreContext)
+  const handleCardOnClick = () => {
+    const originsForClient = origins.forClientSideRendering[releaseBranch].main
+    window.open(`${originsForClient}/a/${trackingArticleSlug}`, '_blank')
```
Comment (Aylie-Chou) [inline src/components/my-reading/article-tracking-card.js:176]: Could we use entity path defined in `@twreporter/core` package to replace hard-coded `/a` ([ref](https://github.com/twreporter/twreporter-npm-packages/blob/d0b553e3d3df415f0be4bcac782669612828f225/packages/core/src/constants/entity-path.js#L2))

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
Mostly looking good~~
plz have a look at comment

File: src/components/my-reading/article-tracking-card.js (hunk 1 above)
Comment (liruchen32) [reply src/components/my-reading/article-tracking-card.js]: fixed

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**